extension/browserUtil.js

```diff
@@ -7,6 +7,10 @@ export async function activeTab() {
   }))[0];
 }
 
+export async function activeTabs() {
```

Comment:
I don't think this function does as advertised: it returns all tabs, not just the active ones.

It would be fine instead to run `browser.tabs.query()` inside the closeAll intent, with whatever options are appropriate there.

Reply:
done!